tests/test_classifier/test_class_prediction_error.py

```diff
@@ -0,0 +1,98 @@
+# tests.test_classifier.test_rocauc
+# Testing for the ROCAUC visualizer
+#
+# Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
+# Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
+# Created:  Tue May 23 13:41:55 2017 -0700
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_rocauc.py [] benjamin@bengfort.com $
+
+"""                                                                                                                  Testing for the ClassPredictionError visualizer
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import pytest
+import numpy as np
+import numpy.testing as npt
+
+from tests.base import VisualTestCase
+from tests.dataset import DatasetMixin
+from yellowbrick.classifier.rocauc import *
+from yellowbrick.exceptions import ModelError
+
+from sklearn.svm import LinearSVC
```

Comment:
For the pyflake tests to pass, we'll have to get rid of all unused imports; that may have been one issue with the tests not passing.

Reply:
done 😄